app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -37,7 +40,15 @@ public void setDateSentByReceiver(LocalDateTime dateSentByReceiver) {
 		this.dateSentByReceiver = dateSentByReceiver;
 	}
 	
-	@Override
+	public LocalDateTime getReceiverDateReceived() {
+        return receiverDateReceived;
+    }
+
+    public void setReceiverDateReceived(LocalDateTime receiverDateReceived) {
+        this.receiverDateReceived = receiverDateReceived;
+    }
+
+    @Override
```

Comment:
Add receiverDateReceived to the display string below

Reply:
I didn't get this comment.

Reply:
Include receiverDateReceived in the value returned by **toString()**